src/app/settings/page.tsx

```diff
@@ -35,31 +37,21 @@ type FormValues = {
   phoneNumber: string;
   gender: (typeof GENDERS)[number];
   industry: (typeof INDUSTRIES)[number] | '';
-  // role: (typeof ROLES)[number];
-  // permissions: (typeof PERMISSIONS)[number];
 };
 
 export default function ProfilePage() {
-  // TODO: get user information from the session - using example data for now
-  const currentUser: FormValues = {
-    firstName: 'John',
-    lastName: 'Doe',
-    email: 'johndoe@email.com',
-    phoneNumber: '+1 (123) 456-7890',
-    gender: 'Male',
-    industry: 'Technology',
-  };
+  const { user } = useAuth();
+  const [currentUser, setCurrentUser] = useState<FormValues | null>(null);
+  const [loading, setLoading] = useState(true);
 
   const form = useForm<FormValues>({
     initialValues: {
-      firstName: currentUser.firstName,
-      lastName: currentUser.lastName,
-      email: currentUser.email,
-      phoneNumber: currentUser.phoneNumber,
-      gender: currentUser.gender,
-      industry: currentUser.industry,
-      // role: "Mentee",
-      // permissions: "announcements",
+      firstName: '',
+      lastName: '',
+      email: '',
+      phoneNumber: '',
+      gender: 'Male',
+      industry: 'Technology',
```

Comment:
these both should be null values too (`''`). don't see why we should set these two as opposed to not setting the rest.